src/Info.ts

```diff
@@ -83,7 +147,23 @@ export default class Info {
             const name = args[1];
             const text = args.splice(2).join(" ");
 
-            message.channel.send(this.addInfo(name, text));
+            let reply = await message.channel.send("What category would you like to put it in?");
+            if (reply instanceof Array) reply = reply[0];
+
+            this.addReactionListener({
+                user: message.author,
+                message: reply,
+                callback: (emoji: Discord.Emoji, listener: ReactionListener) => {
+
+                    if (!this.categories.find(c => emoji.identifier === c.icon))
+                        return; // Not a valid category.
+
+                    message.channel.send(this.addInfo(name, text, emoji));
+                },
+            });
+
+            for (const category of this.categories)
+                await reply.react(category.icon).catch((reason) => console.log(`Cannot react with category '${category}', reason being: ${reason}`));
```

Comment:
Still feel like all control statements should have braces or be on the same line tbh.

Reply:
Meh, it's more the issue that this line is too long and complicated. curly should be able to be enforced with a max length